queries/rmrk/subsquid/collectionsSales.graphql

```diff
@@ -0,0 +1,16 @@
+query collectionsSales($ids: [String!]) {
```

Comment:
since this query is not used only by rmrk I would suggest changing the directory

Reply:
* Added the range buttons
* yep I'm aware that RMRK chain data is still empty. to my understanding I'm querying the correct data so it is what it is (empty)
please have a look at the query and the way i process it  and let me know if you think I'm displaying wrong data

* although not in the requirements, i've added an 'All' time filter button so you could see that under that filter
 RMRK chain data is not empty

* moved the query to @/queries/collectionsSales.graphql

* point to consider:
before my code changes, the volume in top collections and the volume in collection details page did not match
now they do
so either:
1. I fixed a bug
2. I'm calculating wrong volume in top collection, could be the reason why it's empty

Please advice